src/CodeViewer/Viewer.cpp

```diff
@@ -194,4 +300,20 @@ void Viewer::HighlightCurrentLine() {
   setExtraSelections({selection});
 }
 
+int Viewer::WidthPercentageColumn() const {
+  const std::string kWidestPercentage = "100.00 %";
+  return StringWidthInPixels(fontMetrics(), kWidestPercentage);
+}
+
+int Viewer::WidthSampleCounterColumn() const {
+  const std::string kSampleColumnTitle = "Samples";
+  return StringWidthInPixels(fontMetrics(),
+                             kSampleColumnTitle);  // TODO: titles might have smaller font
```

Comment:
Again, are you addressing this in this PR?

Reply:
Yes